test/unittests/test_recordstore.py

```diff
@@ -457,6 +461,9 @@ def test_create_project(self):
     def test_project_info(self):
         self.assertEqual(self.store.project_info("TestProject")["name"], "TestProject")
 
+    def test_clear(self):
+        pass  # raises UserWarning
```

Comment:
Sorry, I forgot to answer that one. It should not cause problems, as upgrading is based on a JSON serialization of the records that is independent of Django.